.gitignore

```diff
@@ -6,3 +6,5 @@ build/
 /node_modules
 /.tmp
 /bin
+.idea/*
```

Comment:
I'd suggest you actually put this in your [global gitignore](https://stackoverflow.com/questions/7335420/global-git-ignore#7335487). Seems pretty specific to your workspace setup.

Reply:
Awesome, didn't know that was a thing. Will update.